server/render/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import ReactDomServer from 'react-dom/server';
+import Helmet from 'react-helmet';
+import memoize from 'lodash/function/memoize';
+import superagent from 'superagent';
+import LruMap from 'lru-map';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+const memoizedRenderToString = memoize( ReactDomServer.renderToString, element => JSON.stringify( element ) );
```

Comment:
We don't have a data cache at the moment, though. I'd prefer some kind of caching that we could always remove later. Also, themes data isn't hugely variable, so I believe there will be some benefit to caching the markup — especially if we can share the markup between node instances.